### ISSUE TITLE
Fix: MD007: hard tabs use case

### ISSUE THREAD

**Issue by Malix-off** [DavidAnson/markdownlint#1010]
# Fix

## Subject

MD007

## Object

When MD010 is disabled (hard tabs use case), make `MD007.indent = 1` and `MD007.start_indent = 1` by default

**Comment by DavidAnson** [comment]
Just because tabs are allowed in a project does not mean they will/should be used for lists. Although I think that could be specified via a parameter to the rule.

**Comment by Malix-off** [comment]
- [Comment Reply](https://github.com/DavidAnson/markdownlint/issues/1010#issuecomment-1766942491) from @DavidAnson:
  > [⋯](https://github.com/DavidAnson/markdownlint/issues/1010#issuecomment-1766942491) because tabs are allowed in a project does not mean they will/should be used for lists [⋯](https://github.com/DavidAnson/markdownlint/issues/1010#issuecomment-1766942491)

  Disagree
  It's a convention to normalize the indentation style project-wise
  If MD010 is disabled, it means that the project is enforcing a tab indentation style
  Tab indentation style => 1 tab / indentation

**Comment by DavidAnson** [comment]
With very few exceptions, rules do not change their behavior based on how other rules are configured. This project prefers spaces and provides rule MD010 to help enforce that. Disabling MD010 does not introduce a requirement to use tabs. As the specification outlines, the behavior of tab stops is not always intuitive (https://spec.commonmark.org/0.30/#tabs).

It sounds like adding the parameter I propose above would satisfy your scenario.

**Comment by Malix-off** [comment]
- [Comment Reply](https://github.com/DavidAnson/markdownlint/issues/1010#issuecomment-1767014752) from @DavidAnson:
  > [⋯](https://github.com/DavidAnson/markdownlint/issues/1010#issuecomment-1767014752) Disabling MD010 does not introduce a requirement to use tabs [⋯](https://github.com/DavidAnson/markdownlint/issues/1010#issuecomment-1767014752)

  Do you mean that disabling MD010 does remove the requirement to not use hard tabs, rather that induce the requirement to use tabs?

**Comment by DavidAnson** [comment]
Correct: MD010 will warn about tabs when it is enabled, but will do nothing when it is disabled.
